registry/storage/driver/s3-aws/s3.go

```diff
@@ -138,6 +140,17 @@ func FromParameters(parameters map[string]interface{}) (*Driver, error) {
 		secretKey = ""
 	}
 
+	backend, ok := parameters["backend"]
+	if !ok {
+		backend = "aws"
+	}
+
+	regionEndpoint, ok := parameters["regionendpoint"]
+	if !ok {
+		regionEndpoint = ""
+	}
+	endpointURL := fmt.Sprint(regionEndpoint)
```

Comment:
i think we need it...i get 'cannot use endpointURL (type interface {}) as type string in field value: need type assertion' if i remove it.